front-end-tools/src/constants.ts

```diff
@@ -4,3 +4,14 @@ import moment from 'moment';
 export const REFRESH_INTERVAL = moment.duration(5, 'seconds');
 
 export const BROWSER_WALLET = ephemeralConnectorType(BrowserWalletConnector.create);
+
+export const DEFAULT_JSON_OBJECT = {
```

Comment:
Can you elaborate on what this is meant for?

Reply:
This is the example JSON object that is shown in the textfield when the user has no `embedded schema` in the module or does not want to use the embedded schema (e.g. if the checkbox "Use module from step 1" is unchecked).

Reply:
Renamed it to:
`EXAMPLE_JSON_OBJECT`